data/Parvej2001.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Parvej Shaikh",
+  "displayStatsPublic" : true,
+  "type":"personal",
+  "bio": "Building Axstudios.tech and Skilled in Full Stack development.Exploring Cloud Native and Blockchain technology with a bachelor of engineering focused in Information Technology",
+  "avatar": "https://github.com/parvej2001.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/p_parvez",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Lets collobrate here",
+      "url": "https://github.com/parvej2001",
+      "icon": "FaGithub"
+    },
+     {
+      "name": "Follow me to hangout and chill",
+      "url": "https://www.instagram.com/parvez_shaikh38",
+      "icon": "FaInstagram"
+    },
```

Comment:
Trailing comma. I think there is no need for it

Reply:
Alright 👍 thanks